### PR TITLE
[T5A1][T13-B1] Nathan Ohun Kwon 

### PR DESCRIPTION
**Pull request by kwonn**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -74,12 +75,37 @@ Use case ends.
...
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook prompts for the tag to change
+3. User inputs the tag
+4. AddressBook prompts for the new tag name
+5. User inputs new tag
+6. AddressBook replaces old tag name with new one for all contacts
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: there is one round of confirmation that should happen between the addressbook and user before changes are made,

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -74,12 +75,37 @@ Use case ends.
...
+Use case ends.
+
+**Extensions**
+
+1a. There are no contacts
+
+> Use case ends
+
+3a. The given tag does not exist
+
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: - what happens when the new tag given is already present in the address book?
- what happens if the new tag is same as the old tag?

File: doc/DeveloperGuide.md [hunk 3]
```diff
@@ -74,12 +75,37 @@ Use case ends.
...
+  Use case resumes at step 2
+
 ## Appendix C : Non Functional Requirements
 
 1. Should work on any [mainstream OS](#mainstream-os) as long as it has Java 8 or higher installed.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should respond to user within 5 seconds.
+6. Should let user export contacts to a text file.
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: i remember your comment during the tutorial, but don't you still think this is a functionality rather than NFR?

**Comment by okkhoy** [comment]
@kwonn 
Some comments added. Please ack & close the PR after reading comments.

**Comment by kwonn** [comment]
Thank you!

**Pull request closed without merging**